### PR TITLE
Update UCP deploy application image

### PR DESCRIPTION
**Pull request by traci-morrison**
### Proposed changes

- https://docs.docker.com/ee/ucp/deploy-application-package/
- Replaces ucp-create-stack.png
- Update screenshot and corresponding text

### Related issues (optional)

- https://docker.atlassian.net/browse/ENGDOCS-291

### CODE REVIEW

**Comment by traci-morrison** [comment]
@KoryKessel-Docker PTAL

File: ee/ucp/deploy-application-package.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ keywords: ucp, swarm, kubernetes, application, app package
 
 Docker Enterprise Edition 2.1 introduces application packages in Docker. With application packages, you can add metadata and settings to an existing Compose file. This gives operators more context about applications they deploy and manage.
 
-An application package can have one of these formats:
+An application package can have one of the following formats:
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change to read, "Application packages can present in one of two different formats, **Directory* or **Single-file**.

File: ee/ucp/deploy-application-package.md [hunk 2]
```diff
@@ -8,7 +8,7 @@ keywords: ucp, swarm, kubernetes, application, app package
...
 
 - **Directory format**: Defined by metadata.yml, a docker-compose.yml, and a settings.yml files inside a `my-app.dockerapp` folder. This is also called the folder format.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Remove the word **format** from the term.

File: ee/ucp/deploy-application-package.md [hunk 3]
```diff
@@ -8,7 +8,7 @@ keywords: ucp, swarm, kubernetes, application, app package
...
 - **Single-file format**: Defined by metadata.yml, docker-compose.yml, and settings.yml concatenated in that order and separated by `---\n` in a single file named named `my-app.dockerapp`.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Remove the word **format** from the term.

File: ee/ucp/deploy-application-package.md [hunk 4]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
 7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+8. If you selected Compose file, enter or upload your `docker-compose.yml` in the **Create Application** window.
 
     ![Provide docker-compose.yml in UCP](/ee/ucp/images/ucp-stack-compose.png)
 
-   or if you selected **App Package**, enter or upload the application package in the single-file format.
+   If you selected **App Package**, enter or upload the application package in the single-file format.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Capitalize 'single'.

File: ee/ucp/deploy-application-package.md [hunk 5]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
...
 
     ![Provide application package in UCP](/ee/ucp/images/ucp-stack-package.png)
 
 9. Select **Create**.
 
 ## Single-file format example
 
-Here is an example of a single-file application package:
+The following file is an example of a single-file application package:
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Capitalize 'single'.

File: ee/ucp/deploy-application-package.md [hunk 6]
```diff
@@ -17,14 +17,14 @@ Once an application package has been deployed, you manipulate and manage it as y
 
 ## Creating a stack in the UCP web interface
 
-To create a stack in the UCP web interface, follow these steps:
+To create a stack in the UCP web interface:
 
 1. Go to the UCP web interface.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change 'Go to' to 'Access'.

File: ee/ucp/deploy-application-package.md [hunk 7]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: For Step 5, change the wording of the second sentence to read: "Note that if you select  **Kubernetes Workloads** the **Namespace** drop-down list will display, from which you must select one of the available namespaces.

File: ee/ucp/deploy-application-package.md [hunk 8]
```diff
@@ -17,14 +17,14 @@ Once an application package has been deployed, you manipulate and manage it as y
...
 
 1. Go to the UCP web interface.
-2. In the lefthand menu, first select **Shared Resources**, then **Stacks**.
+2. From the left menu, select **Shared Resources** > **Stacks**.
 
-    ![Create stacks in UCP](/ee/ucp/images/ucp-create-stack.png)
+    ![Create stacks in UCP](/ee/ucp/images/v32stacks.png)
 
-3. Select **Create Stack** to display **1. Configure Application** in the stack creation dialog.
+3. Select **Create Stack** to open the **Create Application** window.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change wording to read, "Click the **Create Stack** button located at the right of the screen to open the **Create Application** window."

File: ee/ucp/deploy-application-package.md [hunk 9]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
 
 6. Select either **Compose File** or **App Package** for the **Application File Mode**.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change to read, "Click to indidate the **Application File Mode**, either **Compose File** or **App Package**.

File: ee/ucp/deploy-application-package.md [hunk 10]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
...
 7. Select **Next**.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change to read, "Click **Next** to activate **Section 2. Add Application File**."

File: ee/ucp/deploy-application-package.md [hunk 11]
```diff
@@ -35,19 +35,19 @@ To create a stack in the UCP web interface, follow these steps:
...
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+8. If you selected Compose file, enter or upload your `docker-compose.yml` in the **Create Application** window.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Change to read: Add the pertinent application file, according to the selected Application File Mode.

File: ee/ucp/deploy-application-package.md (hunk 4 above)
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Delete (handled above as a bullet point).

File: ee/ucp/deploy-application-package.md (hunk 1 above)
Comment (traci-morrison) [reply ee/ucp/deploy-application-package.md]: Can you clarify this comment? Should

File: ee/ucp/deploy-application-package.md (hunk 5 above)
Comment (traci-morrison) [reply ee/ucp/deploy-application-package.md]: Can you explain this comment in more detail?

**Comment by traci-morrison** [comment]
@KoryKessel-Docker I have a question about one of your suggestions. PTAL.

File: ee/ucp/deploy-application-package.md (hunk 1 above)
Comment (KoryKessel-Docker) [reply ee/ucp/deploy-application-package.md]: My apologies, @traci-morrison. I was going down one path in my edit, changed tracks, and then failed to clean up the half-baked suggestion that I was first making.

I suggest that the leading phrase be entirely deleted in favor of "Application packages can present in one of two different formats, Directory or Single-file." That alone will suffice for leading into the bullet descriptions for the two formats.

Comment (traci-morrison) [reply ee/ucp/deploy-application-package.md]: Thanks, I made that change. PTAL.

File: ee/ucp/deploy-application-package.md [hunk 12]
```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
 
 Docker Enterprise Edition 2.1 introduces application packages in Docker. With application packages, you can add metadata and settings to an existing Compose file. This gives operators more context about applications they deploy and manage.
 
-An application package can have one of these formats:
+Application packages can present in one of two different formats, Directory or Single-file:
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: In my opinion, the two format types should be in bold at first mention, as well as at the head of their bullets directly below: Application packages can present in one of two different formats, **Directory** or **Single-file**:

File: ee/ucp/deploy-application-package.md (hunk 7 above)
Comment (KoryKessel-Docker) [reply ee/ucp/deploy-application-package.md]: Please put **Orchestrator Mode** and **Namespace** in bold (and be sure that Namespace is capitalized, as it is on-screen).

File: ee/ucp/deploy-application-package.md [hunk 13]
```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
...
-1. Go to the UCP web interface.
-2. In the lefthand menu, first select **Shared Resources**, then **Stacks**.
+1. Access the UCP web interface.
+2. From the left menu, select **Shared Resources** > **Stacks**.
 
-    ![Create stacks in UCP](/ee/ucp/images/ucp-create-stack.png)
+    ![Create stacks in UCP](/ee/ucp/images/v32stacks.png)
 
-3. Select **Create Stack** to display **1. Configure Application** in the stack creation dialog.
+3. Click **Create Stack** to open the **Create Application** window. The **1. Configure Application** section will be active.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Please change "...section will be active." to read "...section will become active."

File: ee/ucp/deploy-application-package.md [hunk 14]
```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
...
-5. Select either **Swarm Services** or **Kubernetes Workloads** for the orchestrator mode. If you select Kubernetes, also select a namespace in the **Namespace** drop-down list.
+5. Click to indicate the Orchestrator Mode, either **Swarm Services** or **Kubernetes Workloads**. Note that if you select Kubernetes Workloads, the namespace drop-down list will display, from which you must select one of the namespaces offered. 
 
     ![Specify namespace for a stack in UCP](/ee/ucp/images/ucp-stack-namespace.png)
 
-6. Select either **Compose File** or **App Package** for the **Application File Mode**.
-7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+6. Click to indicate the Application File Mode, either **Compose File** or **App Package**.
+7. Click **Next** to open the **2. Add Application File** section.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Put **Application File Mode** in bold text.

Comment (traci-morrison) [reply ee/ucp/deploy-application-package.md]: I prefer to leave the sentence as written ("to open vs. to activate"). I think "to open" is the better choice when describing actions relating to a GUI.

File: ee/ucp/deploy-application-package.md [hunk 15]
```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
...
 4. Enter a name for the stack in the **Name** field.
-5. Select either **Swarm Services** or **Kubernetes Workloads** for the orchestrator mode. If you select Kubernetes, also select a namespace in the **Namespace** drop-down list.
+5. Click to indicate the **Orchestrator Mode**, either **Swarm Services** or **Kubernetes Workloads**. Note that if you select Kubernetes Workloads, the **Namespace** drop-down list will display, from which you must select one of the namespaces offered. 
 
     ![Specify namespace for a stack in UCP](/ee/ucp/images/ucp-stack-namespace.png)
 
-6. Select either **Compose File** or **App Package** for the **Application File Mode**.
-7. Select **Next**.
-8. If you selected Compose file, enter or upload your `docker-compose.yml` in **2. Add Application File**.
+6. Click to indicate the Application File Mode, either **Compose File** or **App Package**.
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: Make **Application File Mode** bold.

File: ee/ucp/deploy-application-package.md [hunk 16]
```diff
@@ -8,46 +8,46 @@ keywords: ucp, swarm, kubernetes, application, app package
...
-- **Single-file format**: Defined by metadata.yml, docker-compose.yml, and settings.yml concatenated in that order and separated by `---\n` in a single file named named `my-app.dockerapp`.
+- **Directory**: Defined by metadata.yml, a docker-compose.yml, and a settings.yml files inside a `my-app.dockerapp` folder. This is also called the folder format.
+- **Single-file**: Defined by metadata.yml, docker-compose.yml, and settings.yml concatenated in that order and separated by `---\n` in a single file named named `my-app.dockerapp`.
 
 Once an application package has been deployed, you manipulate and manage it as you would any stack.
 
 ## Creating a stack in the UCP web interface
 
-To create a stack in the UCP web interface, follow these steps:
+To create a stack in the UCP web interface:
```
Comment (KoryKessel-Docker) [inline ee/ucp/deploy-application-package.md]: I suggest deleting the redundant lead-in sentence.

**Comment by traci-morrison** [comment]
> A couple more small edits and it should be good to go.

Changes made. Can you please review and approve? Thanks.

**Pull request merged**